### DISCUSSION TITLE
Set up to develop the frontend only

### DISCUSSION THREAD

**Discussion by harrmitch** [mastodon/mastodon#30623] [Q&A]
Hi, I want to hack on the frontend React app in `javascript/mastodon` only and I don't want to go to the trouble of setting up a whole Rails development environment for Mastodon on my Windows machine. Is there any option or guide for me? Thanks.

**Comment by mjankowski** [comment]
Much of the front end working correctly is quite coupled to and dependent on a functioning backend.

Storybook was added recently, so I suppose for some things you could operate in there only?
